samples/places-autocomplete-addressform/src/style.scss

```diff
@@ -87,4 +94,17 @@ input:focus {
 	position:relative;
 	top:1px;
 }
+ 
+.clear-form-text {
+  padding: 6px 0;
+  align-self: flex-end;
+}
+ 
+.clear-form-text:link {
```

Comment:
would it be better to use :hover/:focus instead?

Reply:
Used input type selector instead